### ISSUE TITLE
beginShape's constants are missing

### ISSUE THREAD

**Issue by berinhard** [berinhard/pyp5js#36]
All `beginShape`'s constants are raising: `ReferenceError: LINES is not defined[Learn More]`.

I discovered this while working on an example to show how `beginShape` works ¬¬

To document, the code is here: 

```python
from pytop5js import *


points = []
shape_mode = 0
should_close = 0

def setup():
    createCanvas(window.innerWidth, window.innerHeight)
    noFill()
    strokeWeight(3)

def draw():
    background(242)
    stroke(27)

    beginShape()
    for x, y in points:
        vertex(x, y)
    endShape(CLOSE)


def mouseClicked():
    points.append((
        mouseX, mouseY
    ))


def keyPressed():
    global shape_mode
    modes_map = {
        '1': POINTS,
        '2': LINES,
        '3': TRIANGLES,
        '4': TRIANGLES_STRIP,
        '5': TRIANGLE_FAN,
        '6': QUADS,
        '7': QUAD_STRIP,
    }

    try:
        shape_mode = modes_map[key]
    except KeyError:
        pass


# ==== This is required by pyp5js to work

# Register your events functions here
event_functions = {
    "keyPressed": keyPressed,
    'mouseClicked': mouseClicked,
}
start_p5(setup, draw, event_functions)
```

**Comment by berinhard** [comment]
It's practically the same work as in 15712395a3de5d0c5d32d7470129c3e31e4cbbfc

**Comment by berinhard** [comment]
This will lead to a greater work which is: mapping all missing constants and add them to [`pyp5js/assets/p5_reference.yml`](https://github.com/berinhard/pyp5js/blob/develop/pyp5js/assets/p5_reference.yml). 

This is required because, when I added the first variables and constants, I based myself on the [p5.js's reference page](https://p5js.org/reference/). But the thing is that there are constants such as `WEBGL`, `HSB` or `TRIANGLES` that are only used as arguments to specific methods (`createCanvas`, `colorMode` and `beginShape` and, because of that, they aren't listed in the reference manual, but inside their specific function manual page ([here's an example](https://p5js.org/reference/#/p5/beginShape)).

To discover all the missing constants, we'll have to open method by method and document them at this task so we can tackle all of them in a single PR. @villares if you have time, could you help me with that? I also think, once we have the constants list, it'll be the time to introduce the refactor you've suggested: to separate the constants from the variables list and do not reload them o `pre_draw`.

**Comment by berinhard** [comment]
Missing constants so far:

**Comment by berinhard** [comment]
All the docs are documented in ths [`constants.js` file](https://github.com/processing/p5.js/blob/master/src/core/constants.js).

**Comment by berinhard** [comment]
Fixed by #39
